### PR TITLE
Update the OS version definition for each server

### PR DESCRIPTION
**Pull request by jouvin**
The changes in this PR fix the problem identified in https://github.com/quattor/template-library-core/pull/237 after the structure change of grid/OS repositories (`main` branch instead of one branch per major version). The problem was caused by the fact that the (obsolete) default OS version is not part of the `main` branch.

### CODE REVIEW

**Comment by jrha** [comment]
Ah, failing due to https://github.com/quattor/configuration-modules-core/issues/1787

**Comment by jouvin** [comment]
The remaining error is caused by https://github.com/quattor/configuration-modules-core/issues/1787. I'm reluctant to workaround the problem in the affected profiles. 

The problem comes from that the fact `systemd/legacy/chkconfig` is loaded as part of the OS configuration but no systemd unit is defined in the systemd component for some profiles, which is a valid use case.

File: sites/example/site/os/version_db.pan [hunk 1]
```diff
@@ -24,22 +26,25 @@ variable YUM_OS_DISTRIBUTION = {
...
-
-    escape("nagios-server.example.org"), "el7x-x86_64",
-    escape("nagios-master.example.org"), "el7x-x86_64",
-    escape("nagios-slave-A.example.org"), "el7x-x86_64",
-    escape("nagios-slave-B.example.org"), "el7x-x86_64",
+    # Grid cluster
+    '{apel.example.org}', 'el7x-x86_64',
+    '{argus.example.org}', 'el7x-x86_64',
+    '{bdii.example.org}', 'el7x-x86_64',
+    '{px.example.org.tpl.disabled}', 'el7x-x86_64',
```
Comment (jrha) [inline sites/example/site/os/version_db.pan:33]: Is this right?

Comment (jouvin) [reply sites/example/site/os/version_db.pan]: Good catch! Fixed

**Comment by jrha** [comment]
That does remind me that I'd like to discuss options for not having `template-libary-core` be a mix of manually maintained and automatically generated content.

**Comment by jrha** [comment]
I suppose manually fixing the schema for now is fine.

**Comment by jouvin** [comment]
> That does remind me that I'd like to discuss options for not having `template-libary-core` be a mix of manually maintained and automatically generated content.

I agree it is not ideal but I don't have any idea on how avoiding it. Except moving the manually maintained content in another repo and importing it the same way as we do for configuration module templates. But at the price that it is much more difficult to test modifications in example profiles, as we do in `test-library` script, as this script rely on `template-library-core` and it may be unnecessarily complex to try to have it merging modifications from another repo... but may be I'm missing something...

**Comment by jrha** [comment]
Indeed, I don't see any obviously good options.

**Review by jrha — APPROVED** [review]

**Pull request merged**